source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,187 @@
+.. _install-configfile:
+
+Install using clr-installer and a configuration file
+####################################################
+
+This guide describes how to install |CL-attr| using the clr-installer tool
+with a configuration file. The configuration file (:file:`clr-installer.yaml`)
+can be used to duplicate the same installation configuration on more machines.
+
+
+System requirements
+*******************
+
+Ensure that your target system supports the installation:
+
+* :ref:`system-requirements`
+* :ref:`compatibility-check`
+
+Process
+*******
+
+This guide describes two methods for using a configuration file with the
+clr-installer tool. You can use either method to achieve the same goal. Choose
+the method that works best for your setup.
+
+If you are installing |CL| for the first time, we recommend Example 1.
+
+To clone an existing |CL| setup on another system, we recommend Example 2.
+
+Example 1
+=========
+
+This method uses an existing configuration file template to perform a new
+installation.
+
+Perform the following steps:
+
+#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.

```

Comment:
@MCamp859 - pls change line 38 to "Get the latest Clear Linux OS installer image from the downloads page. Look for the clear-[version number]-live-server.iso file."  Pls make a link to "downloads page" to "https://clearlinux.org/downloads"